certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -2168,6 +2168,25 @@
             "x-ms-visibility": "advanced",
             "type": "integer"
           },
+          {
+            "name": "signatureType",
+            "in": "query",
+            "description": "DocuSign standards-based-signature (SBS) types.",
+            "x-ms-summary": "Signature type",
+            "x-ms-visibility": "advanced",
+            "type": "string",
+            "enum": [
+              "DS Electronic (SES)",
+              "DS EU Advanced (AES)",
+              "DS EU Qualified (QES)"
+            ],
+            "x-ms-dynamic-values": {
```

Comment:
It seems unusual to have an enum and dynamic values for the same param. 
Which one are you intending to keep

Reply:
Removed enum. It was not required.